internal/provider/resource_ko_image.go

```diff
@@ -40,6 +40,13 @@ func resourceImage() *schema.Resource {
 				},
 				ForceNew: true, // Any time this changes, don't try to update in-place, just create it.
 			},
+			"working_dir": {
+				Description: "working directory for the build",
```

Comment:
Nit: I think we could document a bit on how relative versus absolute paths behave here to help folks understand how this is all going to go down. Like if `importpath` is `./foo/cmd` and `working_dir` is `./bar/baz`, am I going to build `bar/baz/foo/cmd`?

From the code it looks to me like `working_dir` is relative to the terraform process current working directory and `importpath` is relative to `working_dir`, is that correct?

Reply:
> Awesome. Supprised at how easy the implementation. Just a nit about documenting how the relative paths work for folks

Oh that's because it doesn't work 🙃